### PR TITLE
Add generators for forms and decorators

### PR DESCRIPTION
**Pull request by haukot**

### CODE REVIEW

**Comment by haukot** [comment]
Please check TODOs in code

File: README.md [hunk 1]
```diff
@@ -88,31 +88,37 @@ Tramway::Admin.navbar_structure(
   project: :your_application_name
 )
 ```
+#### 9. Create decorators and forms for models
```
Comment (kalashnikovisme) [inline README.md:91]: need to be added `for all available_models`

File: lib/tramway/admin/generators/install_generator.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+module Tramway
+  module Admin
+    module Generators
+      class InstallGenerator < ::Rails::Generators::Base
+        source_root File.expand_path('templates', __dir__)
+
+        def run_decorator_generators
+          ::Tramway::Admin.available_models_for(project).map do |model|
+            generate 'tramway:admin:model', model.to_s
```
Comment (kalashnikovisme) [inline lib/tramway/admin/generators/install_generator.rb:12]: `Tramway::User::User.role.values`

Comment (kalashnikovisme) [reply lib/tramway/admin/generators/install_generator.rb:12]: "tramway:#{current_user.role}:model"

File: lib/tramway/admin/generators/model_generator.rb [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+          string: :string,
+          uuid: :string,
+          float: :number,
+          integer: :number,
+          # TODO: which types should be these?
+          # boolean: ,
+          # enum: "Field::Select",
+        }
+        DEFAULT_FIELD_TYPE = :string
+        READ_ONLY_ATTRIBUTES = %w[id created_at updated_at]
```
Comment (kalashnikovisme) [inline lib/tramway/admin/generators/model_generator.rb:23]: add `uuid` here please

**Review by kalashnikovisme — CHANGES_REQUESTED** [review]

File: lib/tramway/admin/generators/install_generator.rb (hunk 2 above)
Comment (haukot) [reply lib/tramway/admin/generators/install_generator.rb]: fixed with argument for generator(see Readme)

**Comment by haukot** [comment]
@kalashnikovisme Updated

File: lib/tramway/admin/generators/model_generator.rb [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+          elsif relationship.collection?
+            :has_many
+          elsif relationship.polymorphic?
+            :polymorphic
+          else
+            :belongs_to
+          end
+        end
+
+        def column_type_for_attribute(attr)
```
Comment (kalashnikovisme) [inline lib/tramway/admin/generators/model_generator.rb:95]: Let's remove this `enum` content 😁

**Review by kalashnikovisme — CHANGES_REQUESTED** [review]

**Comment by kalashnikovisme** [comment]
@haukot one last update, please!!! :)

**Review by kalashnikovisme — CHANGES_REQUESTED** [review]
One last update pleaase!!!!) :)

**Comment by haukot** [comment]
@kalashnikovisme updated :)

**Comment by kalashnikovisme** [comment]
Let's merge this shit :)

**Comment by haukot** [comment]
Watch your tongue, young man!

**Comment by haukot** [comment]
@kalashnikovisme I don't have the access to merge it. Just saying

**Comment by kalashnikovisme** [comment]
DONE!

**Comment by haukot** [comment]
🎉

**Pull request merged**